server/database/config.js

```diff
@@ -1,13 +1,29 @@
 var Sequelize = require('sequelize');
 
-var Messages = Sequelize.define('Messages', {
+var dbconfig = {};
+dbconfig.database = process.env.database || 'crew';
+dbconfig.username = process.env.username || 'root';
+dbconfig.password = process.env.password || '';
+dbconfig.hostname = process.env.hostname || 'localhost';
+
+var orm = new Sequelize(dbconfig.database, dbconfig.username, dbconfig.password, {
+  host: dbconfig.hostname
+});
+
+var Messages = orm.define('messages', {
   message: Sequelize.TEXT,
-  timestamp: Sequelize.DATE,
+  name: Sequelize.STRING,
```

Comment:
comma at the end of this, we need to decided as a group @pravipati whats the verdict on objects last comma?